src/ray/stats/metric_exporter_client.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RAY_METRIC_EXPORTER_CLIENT_H
+#define RAY_METRIC_EXPORTER_CLIENT_H
```

Comment:
Use `pragma once` instead.

Reply:
This doesn't seem to be resolved.

Reply:
> This doesn't seem to be resolved.

Thank you for reminding. It's removed in lastest commit.